tests/test_diffraction_vectors.py

```diff
@@ -21,61 +21,3 @@
 from hyperspy.signals import Signal1D, Signal2D
 from pyxem.signals.electron_diffraction import ElectronDiffraction
 from pyxem.signals.diffraction_vectors import DiffractionVectors
-
```

Comment:
This is an intentional deletion?

Reply:
I think this file had some tests that didn't test diffraction vectors in it? so I deleted it as a waste of time

Reply:
Sounds fine. Won't do anything more with this file in this PR